runtime/compiler/trj9/runtime/IProfiler.cpp

```diff
@@ -2789,15 +2798,7 @@ TR_IPBCDataCallGraph::setData(uintptrj_t v, uint32_t freq)
             }
          }
       else
```

Comment:
Please put curly brackets even if it's just one line. Our rules say that if the 'then' part has curly brackets, the 'else' part must have as well for symmetry.